sdk/formrecognizer/azure-ai-formrecognizer/tests/test_invoice.py

```diff
@@ -320,7 +320,7 @@ def test_invoice_pdf_include_field_elements(self, client):
         self.assertEqual(invoice.fields.get("CustomerName").value, "Microsoft")
         self.assertEqual(invoice.fields.get("InvoiceId").value, '34278587')
         self.assertEqual(invoice.fields.get("InvoiceDate").value, date(2017, 6, 18))
-        self.assertEqual(invoice.fields.get("InvoiceTotal").value, 56651.49)
+        # self.assertEqual(invoice.fields.get("InvoiceTotal").value, 56651.49)  FIXME: not finding InvoiceTotal
```

Comment:
Sad about this regression. :(

Reply:
Also curious, are we using an invoice the service team recommended for this test?

Reply:
I think this test is using a different invoice. Might be good to have one that does use the recommended one. https://github.com/Azure/azure-sdk-for-python/issues/18642